packages/analytics-types/src/config.ts

```diff
@@ -77,16 +77,24 @@ export interface ReactNativeTrackingOptions extends TrackingOptions {
 
 export interface AdditionalBrowserOptions {
   attribution?: AttributionBrowserOptions;
+  trackPageViews?: boolean | PageTrackingBrowserOptions;
 }
 
 export interface AttributionBrowserOptions {
   disabled?: boolean;
   excludeReferrers?: string[];
   initialEmptyValue?: string;
+  resetSessionOnNewCampaign?: boolean;
   trackNewCampaigns?: boolean;
   trackPageViews?: boolean;
```

Comment:
Recommend we add some deprecation jsdoc comments here
```suggestion
  /** @deprecated New campaigns are now always tracked by default. Setting this to true also sets resetSessionOnNewCampaign to true. */
  trackNewCampaigns?: boolean;
  /** @deprecated Use the new top level trackPageViews configuration instead. */
  trackPageViews?: boolean;
```